### ISSUE TITLE
Wither Despawns when Unloaded

### ISSUE THREAD

**Issue by kakduman** [pl3xgaming/Purpur#1357]
### Spark link

https://spark.lucko.me/njOTSLi67G

### Expected behavior

Wither should still exist instead of despawning. The issue does not occur on Pufferfish and Paper.

### Observed/Actual behavior

The wither despawned.

### Steps/models to reproduce

1. new server with only latest purpur
2. tp to random location (e.g. 1000 100 1000)
3. summon wither & nametag wither (optionally cage it to ensure it can't escape)
4. return to 0 100 0 to unload chunk
5. return to 1000 100 1000
6. observe missing wither.

### Purpur version

Current: git-Purpur-1996 (MC: 1.20.1)*
Previous: git-Pufferfish-5 (MC: 1.20.1)
* You are running the latest version

### Agreements

- [X] I am running the latest version of Purpur available from https://purpurmc.org/downloads.
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [ ] I ticked all the boxes without actually reading them
- [X] My version of Minecraft is supported by Purpur.

### Other

I believe that this issue is an expansion/duplicate of #1355, but that issue seems hard to follow and doesn't get to the core of the bug.

**Comment by Codzsters** [comment]
This has also happened on my server. 

Seems to only have been a recent thing since 1.20. Withers did not despawn in 1.19.4 versions (to the best of my memory), and since our world has updated all withers have despawned and continue to despawn when unloading chunks. (We update to the latest build of Purpur whenever it becomes available)

If I remember correctly, vanilla behaviour would suggest you do not need to rename a wither to stop it despawning (although I could be wrong on this)
